packages/flutter/lib/src/widgets/container.dart

```diff
@@ -391,7 +391,7 @@ class Container extends StatelessWidget {
       );
     }
 
-    if (alignment != null)
+    else if (alignment != null)
```

Comment:
can you please add `{ }` around the body of the `if` and `else if` to make this a little more readable?

Reply:
Thanks!